src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -718,10 +718,7 @@ BookStep<TIn, TOut, TDerived>::revImp (
             // subtracting them leaves a result of zero. This can cause the check for
             // (stpAmt.out > remainingOut) to incorrectly think an offer will be funded
             // after subtracting remainingIn.
-            if (fix1298(sb.parentCloseTime()))
-                return offer.fully_consumed();
-            else
-                return false;
+            return offer.fully_consumed();
```

Comment:
Since you're here, I'm really confused about this comment. What is it telling me that's relevant _here_? Is this an artifact of older code?

Reply:
I wrote that comment. It is still relevant. I agree it's not very clear. Is this clearer? 
```
            // Given that (stpAmt.out > remainingOut), it's natural to expect
            // that the offer will still be funded after consuming remainingOut.
            // However, this is not always the case. When the mantissas of two
            // iou amounts differ by less than ten, then subtracting them leaves
            // a result of zero. So instead of declaring the offer is still funded,
            // we need to check explicitly with a call to offer.fully_consumed
            // (see RIPD-1298)
```

Reply:
Thank you, that's better. Consider instead:

```
// Explicitly check whether an offer is funded: given that we have
// (stpAmt.out > remainingOut), it's natural to expect the offer
// will still be funded after consuming remainingOut but that is
// not always the case: if the mantissas of two IOU amounts differ
// by less than ten, then subtracting them result in a zero.
```

Reply:
Yep, that's better. Thanks @nbougalis . Let's go with that ^^^.